GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -78,15 +78,29 @@ class AccountDetailsViewController: UIViewController {
             showErrorAlert()
             return nil
         } else {
-            let account = MessengerAccount(deploymentId: deploymentIdTextField.text ?? "",
+            var account = MessengerAccount(deploymentId: deploymentIdTextField.text ?? "",
                                            domain: domainIdTextField.text ?? "",
                                            logging: loggingSwitch.isOn)
             
+            
+            account = addCustomAttributes(account: account)
+            
             updateUserDefaults()
             return account
         }
     }
     
+    private func addCustomAttributes(account: MessengerAccount) -> MessengerAccount {
```

Comment:
the account is passed by reference to the method, right?
If so why you need to return it as well?